lona/pytest.py

```diff
@@ -96,3 +99,56 @@ def setup_aiohttp_app(loop: AbstractEventLoop) -> Application:
         )
 
     return setup_lona_project_context
+
+
+def eventually(timeout: float = 5, interval: float = 1) -> Iterator[AsyncContextManager]:  # NOQA: LN001
+    """
+        Wait for expected state in async test.
```

Comment:
The comment don't has to be indented this far

Reply:
Fixed